engine/dlib/src/dlib/connection_pool.cpp

```diff
@@ -208,13 +208,13 @@ namespace dmConnectionPool
 
     static void DoClose(HPool pool, Connection* c)
     {
+        if (c->m_SSLSocket != dmSSLSocket::INVALID_SOCKET_HANDLE) {
```

Comment:
Vart sätts INVALID_SOCKET_HANDLE när man delete'ar sslsocketen?

Reply:
It's set when c->Clear() is called:

https://github.com/defold/defold/blob/dev/engine/dlib/src/dlib/connection_pool.cpp#L43-L49

https://github.com/defold/defold/blob/dev/engine/dlib/src/dlib/connection_pool.cpp#L218